Src/AutoFixtureUnitTest/RangedNumberGeneratorTest.cs

```diff
@@ -166,6 +167,35 @@ public void CreateReturnsCorrectResultWhenRunOutOfNumbers(int minimum, int maxim
             loopTest.Execute(loopCount, expectedResult);
             // Teardown
         }
+        
+        [Theory]
+        [InlineData(typeof(byte))] 
+        [InlineData(typeof(sbyte))] 
+        [InlineData(typeof(short))]  
+        [InlineData(typeof(ushort))]
+        [InlineData(typeof(int))]  
+        [InlineData(typeof(uint))]  
+        [InlineData(typeof(long))] 
+        [InlineData(typeof(ulong))] 
+        [InlineData(typeof(decimal))]
+        [InlineData(typeof(float))]  
+        [InlineData(typeof(double))]
+        public void ReturnsSameNumberWhenMinimumAndMaximumAreSame(Type requestType)
+        {
+            // Fixture setup
+            var range = Convert.ChangeType(42, requestType, CultureInfo.InvariantCulture);
```

Comment:
Perhaps, we could even parameterize the `42` magic number, but not that big of a deal.

Reply:
Cannot, because no way to specify a constant for the `decimal` type.